lib/renderer.js

```diff
@@ -1122,11 +1122,14 @@
             if (!layer || layer.visible === false || layer.group && layer.group.visible === false) {
                 return false;
             }
+
             var hasEffect = layer.layerEffects && (
-                (layer.layerEffects.bevelEmboss && layer.layerEffects.bevelEmboss.enabled) ||
-                (layer.layerEffects.dropShadow && layer.layerEffects.dropShadow.enabled) ||
-                (layer.layerEffects.outerGlow && layer.layerEffects.outerGlow.enabled) ||
-                (layer.layerEffects.frameFX && layer.layerEffects.enabled));
```

Comment:
wow.. just noticed the typo on old line 1129 --- we should have been checking layer.layerEffects.frameFX.enabled, instead of layer.layerEffects.enabled!

New changes look like a good improvement.